api/proto/info/info.proto

```diff
@@ -406,7 +406,7 @@ message ZInfoDevice {
   repeated ZInfoDeviceTasks tasks = 37;
 
   bool maintenance_mode = 38;
-  MaintenanceModeReason maintenance_mode_reason = 39;
+  repeated MaintenanceModeReason maintenance_mode_reason = 39;
```

Comment:
I don't think this change is backwards compatible in protobuf. Can you verify?
If not we need to pick a new tag for the repeated and have the controllers handle both.